scripts/symlink-ee.mjs

```diff
@@ -0,0 +1,23 @@
+import fs from 'fs';
+import path from 'path';
+import { execSync } from 'child_process';
+
+// Get the current directory
+const currentDir = process.cwd();
+
+// Remove the prefix up to and including "enterprise/packages/"
+const relativePath = currentDir.split('enterprise/packages/')[1];
+
+// Count the number of slashes in the relative path to determine the depth
+const depth = (relativePath.match(/\//g) || []).length;
+
+// Generate the correct number of "../" for the symlink command
+const prefix = '../'.repeat(depth + 3);
+
+// Generate the symlink command
+const symlinkCommand = `ln -sf ${prefix}.source/${relativePath}/src`;
```

Comment:
we don't miss ` ./src` in the end of the string?
```suggestion
const symlinkCommand = `ln -sf ${prefix}.source/${relativePath}/src ./src`;
```

Reply:
This resolves the nesting issue.